python/social_bot/teacher.py

```diff
@@ -0,0 +1,128 @@
+# Copyright (c) 2019 Horizon Robotics. All Rights Reserved.
+
+from abc import abstractmethod
+import numpy as np
+import random
+
+
+class TeacherAction(object):
```

Comment:
a bit misleading as it has nothing to do with action, perhaps better name TeacherResponse?

Reply:
I think it's ok to use TeacherAction